# HG changeset patch
# User Rik <rik@octave.org>
# Date 1497568406 25200
#      Thu Jun 15 16:13:26 2017 -0700
# Branch stable
# Node ID ff6bcf3d7e1fbaf6bc0bddd69587c801c62723d8
# Parent  afacc2f244bb957d3a2742dc6cf81a1b86487de5
playblocking.m: Correct documentation about start and limits inputs (bug #51217).

* playblocking.m: Document that start and limit refer to the sample number, not
to the number of seconds.

diff --git a/scripts/audio/@audioplayer/playblocking.m b/scripts/audio/@audioplayer/playblocking.m
--- a/scripts/audio/@audioplayer/playblocking.m
+++ b/scripts/audio/@audioplayer/playblocking.m
@@ -17,19 +17,19 @@
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {} {} playblocking (@var{player})
 ## @deftypefnx {} {} playblocking (@var{player}, @var{start})
 ## @deftypefnx {} {} playblocking (@var{player}, @var{limits})
 ## Play audio stored in the audioplayer object @var{player} with blocking.
 ##
-## Given optional argument start, begin playing at @var{start} seconds in the
+## Given optional argument start, begin playing at @var{start} samples in the
 ## recording.  Given a two-element vector @var{limits}, begin and end playing
-## at the number of seconds specified by the elements of the vector.
+## at the number of samples specified by the elements of the vector.
 ## @end deftypefn
 
 function playblocking (varargin)
 
   if (nargin < 1 || nargin > 2)
     print_usage ();
   endif
 
